release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -129,16 +129,70 @@ You can now move etcd from a root volume (Cinder) to a dedicated ephemeral local
+
+* xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Installing a cluster with on {ibm-z-title} and {ibm-linuxone-title}]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
+[discrete]
+==== {ibm-z-title} and {ibm-linuxone-title} notable enhancements
+
+The {ibm-z-name} and {ibm-linuxone-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components and concepts.
+
+This release introduces support for the following features on {ibm-z-name} and {ibm-linuxone-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* `s390x` control plane with `x86_64` multi-architecture compute nodes
+
+[discrete]
+==== Installing a cluster with an LPAR on {ibm-z-title} and {ibm-linuxone-title}
+
+{product-title} now supports user-provisioned installation of {product-title} 4.15 with a logical partition (LPAR) on {ibm-z-title} and {ibm-linuxone-title}.
+
+For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z-lpar.adoc#installing-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name}]
+
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-lpar.adoc#installing-restricted-networks-ibm-z-lpar[Installing a cluster in an LPAR on {ibm-z-name} and {ibm-linuxone-name} in a restricted network]
 
 [id="ocp-4-15-ibm-power"]
 === {ibm-power-title}
 
+{ibm-power-name} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on {ibm-power-name}]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on {ibm-power-name} in a restricted network]
+
+[IMPORTANT]
+====
+Compute nodes must run {op-system-first}.
+====
+
 [discrete]
 ==== {ibm-power-title} notable enhancements
 
+The {ibm-power-name} release on {product-title} {product-version} adds improvements and new capabilities to {product-title} components.
+
+This release introduces support for the following features on {ibm-power-name}:
+
+* Agent-based Installer
+* {cert-manager-operator}
+* {ibm-power-name} Virtual Server Block CSI Driver Operator
+* Installer-provisioned Infrastructure Enablement for {ibm-power-name} Virtual Server
+* Multi-architecture Power Control Plane with support of Intel and {ibm-power-name} workers
+* nx-gzip for Power10 (Hardware Acceleration)
+* openshift-install to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```

Comment:
Markup `openshift-install` and add something like utility/command/binary/tool after:
```suggestion
* The `openshift-install` utility to support various SMT levels on {ibm-power-name} (Hardware Acceleration)
```
I couldn't find a definitive answer for what to call it, but those were some options from around the repo.